SmartDeviceLink/public/SDLMenuCell.h

```diff
@@ -55,6 +55,11 @@ typedef void(^SDLMenuCellSelectionHandler)(SDLTriggerSource triggerSource);
  */
 @property (strong, nonatomic, readonly, nullable) SDLMenuLayout submenuLayout;
 
+/**
+ Primary text of the cell to be displayed on the module. Used to distinguish cells with the same `title` but other fields are different. This is autogenerated by the screen manager. This will not be used when connected to modules supporting RPC 7.1+.
```

Comment:
```suggestion
 Primary text of the cell to be displayed on the module. Used to distinguish cells with the same `title` but other fields are different. This is autogenerated by the screen manager. This will not be used when connected to modules supporting RPC 7.1+ because duplicate titles are supported.
```